packages/react-components/src/entities/EventSidebar/details/Summaries.js

```diff
@@ -98,21 +98,33 @@ export function Summaries({ event, setActiveEvent, addToSearch, addEventTypeToSe
     const mofHierarchyJoined = data?.mofResults.facet.eventTypeHierarchyJoined.sort(function (a, b) {
       return a.key.length - b.key.length
     });
+    //Calculate how may measurements attached to 'Survey','Sample','Find' etc
+    const measurements = mofHierarchyJoined.reduce(function(measurementsCount, record){
+      let total = record.events?.facet?.measurementOrFactTypes.reduce(function(count, mft){
+          count +=mft['count'];
```

Comment:
Why not just `mft.count` ?

Reply:
Looks like this file needs formatting. E.g. `count +=mft['count'];` doesn't have a space after `=`

Reply:
> [4150c41](https://github.com/gbif/gbif-web/commit/4150c414873ab149afbd48fb29bb053ff92cd7d5) `formatter: (value, item) => <>{value || [].join(', ')}</>`
> 
> that looks wrong to me. Doesn't it say: return the array OR an empty string (joining an empty array).

The value should return a list or could be null. It aims to return a joined string, and a blank string when the value is null

Reply:
Done